### PR TITLE
Add UXL identity to README

### PR DESCRIPTION
**Pull request by omalyshe**
### Description 
Add UXL logo, links and Governance section.
Also add a badge for Discussions.

Fixes # - _issue number(s) if exists_

- [x] - git commit message contains an appropriate signed-off-by string _(see [CONTRIBUTING.md](https://github.com/oneapi-src/oneTBB/blob/master/CONTRIBUTING.md#pull-requests) for details)_

### Type of change

_Choose one or multiple, leave empty if none of the other choices apply_

_Add a respective label(s) to PR if you have permissions_

- [ ] bug fix - _change that fixes an issue_
- [ ] new feature - _change that adds functionality_
- [ ] tests - _change in tests_
- [ ] infrastructure - _change in infrastructure and CI_
- [x] documentation - _documentation update_

### Tests

- [ ] added - _required for new features and some bug fixes_
- [x] not needed

### Documentation

- [ ] updated in # - _add PR number_
- [ ] needs to be updated
- [x] not needed

### Breaks backward compatibility
- [ ] Yes
- [x] No
- [ ] Unknown

### Notify the following users
_List users with `@` to send notifications_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +19,7 @@ The library differs from typical threading packages in the following ways:
 
 Refer to oneTBB [examples](examples) and [samples](https://github.com/oneapi-src/oneAPI-samples/tree/master/Libraries/oneTBB) to see how you can use the library.
 
-oneTBB is a part of [oneAPI](https://oneapi.io). The current branch implements version 1.1 of oneAPI Specification.
+oneTBB is a part of [UXL Foundation](http://www.uxlfoundation.org) and is an implementation of [oneAPI specification](https://oneapi.io).
```
Comment (aepanchi) [inline README.md:22]:
```suggestion
oneTBB is a part of the [UXL Foundation](http://www.uxlfoundation.org) and is an implementation of [oneAPI specification](https://oneapi.io).
```

File: README.md [hunk 2]
```diff
@@ -39,6 +40,18 @@ See [Release Notes](RELEASE_NOTES.md) and [System Requirements](SYSTEM_REQUIREME
 ## Installation 
 See [Installation from Sources](INSTALL.md) to learn how to install oneTBB. 
 
+## Governance
+
+The oneTBB project is governed by the UXL Foundation and you can get involved in this project in multiple ways.
```
Comment (aepanchi) [inline README.md:45]:
```suggestion
The oneTBB project is governed by the UXL Foundation. 
You can get involved in this project in following ways:
```

File: README.md [hunk 3]
```diff
@@ -39,6 +40,18 @@ See [Release Notes](RELEASE_NOTES.md) and [System Requirements](SYSTEM_REQUIREME
 ## Installation 
 See [Installation from Sources](INSTALL.md) to learn how to install oneTBB. 
 
+## Governance
+
+The oneTBB project is governed by the UXL Foundation and you can get involved in this project in multiple ways. 
+It is possible to join the [Language Special Interest Group (SIG)](https://github.com/uxlfoundation/foundation/blob/main/language) meetings
```
Comment (aepanchi) [inline README.md:46]:
```suggestion
* Join the [Language Special Interest Group (SIG)](https://github.com/uxlfoundation/foundation/blob/main/language) meetings
```

File: README.md [hunk 4]
```diff
@@ -39,6 +40,18 @@ See [Release Notes](RELEASE_NOTES.md) and [System Requirements](SYSTEM_REQUIREME
...
+where the groups discuss and demonstrate work using this project. Members can also join the Open Source and Specification Working Group meetings.
```
Comment (aepanchi) [inline README.md:47]:
```suggestion
to discuss and demonstrate work using this project. 
* Join the Open Source and Specification Working Group meetings.
```

Comment (aepanchi) [reply README.md:47]: Not clear how users can join open source and specification meetings. Is it also a part of the SIG meetings?

File: README.md [hunk 5]
```diff
@@ -39,6 +40,18 @@ See [Release Notes](RELEASE_NOTES.md) and [System Requirements](SYSTEM_REQUIREME
...
+
+You can also join the mailing lists for the [UXL Foundation](https://lists.uxlfoundation.org/g/main/subgroups) to be informed of when meetings
```
Comment (aepanchi) [inline README.md:49]:
```suggestion
* Join the mailing lists for the [UXL Foundation](https://lists.uxlfoundation.org/g/main/subgroups) to receive meetings schedule and latest updates. 
```

File: README.md [hunk 6]
```diff
@@ -39,6 +40,18 @@ See [Release Notes](RELEASE_NOTES.md) and [System Requirements](SYSTEM_REQUIREME
...
+are happening and receive the latest information and discussions.
```
Comment (aepanchi) [inline README.md:50]:
```suggestion

```

File: README.md [hunk 7]
```diff
@@ -39,6 +40,18 @@ See [Release Notes](RELEASE_NOTES.md) and [System Requirements](SYSTEM_REQUIREME
 ## Installation 
 See [Installation from Sources](INSTALL.md) to learn how to install oneTBB. 
 
+## Governance
+
+The oneTBB project is governed by the UXL Foundation and you can get involved in this project in multiple ways. 
+It is possible to join the [Language Special Interest Group (SIG)](https://github.com/uxlfoundation/foundation/blob/main/language) meetings
+where the groups discuss and demonstrate work using this project. Members can also join the Open Source and Specification Working Group meetings.
+
```
Comment (aepanchi) [inline README.md:48]:
```suggestion
```

File: README.md [hunk 8]
```diff
@@ -39,6 +40,18 @@ See [Release Notes](RELEASE_NOTES.md) and [System Requirements](SYSTEM_REQUIREME
...
+You can also join the mailing lists for the [UXL Foundation](https://lists.uxlfoundation.org/g/main/subgroups) to be informed of when meetings
+are happening and receive the latest information and discussions.
+
```
Comment (aepanchi) [inline README.md:51]:
```suggestion
```

File: README.md [hunk 9]
```diff
@@ -39,6 +40,18 @@ See [Release Notes](RELEASE_NOTES.md) and [System Requirements](SYSTEM_REQUIREME
...
+You can contribute to this project and also contribute to the specification for this project, read the [CONTRIBUTING](./CONTRIBUTING.md) page
```
Comment (aepanchi) [inline README.md:52]:
```suggestion
* Contribute to oneTBB project or oneTBB specification. Read [CONTRIBUTING](./CONTRIBUTING.md)
```

File: README.md [hunk 10]
```diff
@@ -42,14 +42,13 @@ See [Installation from Sources](INSTALL.md) to learn how to install oneTBB.
...
-are happening and receive the latest information and discussions.
-
-You can contribute to this project and also contribute to the specification for this project, read the [CONTRIBUTING](./CONTRIBUTING.md) page
+The oneTBB project is governed by the UXL Foundation. 
+You can get involved in this project in following ways:
+* Join the [Language Special Interest Group (SIG)](https://github.com/uxlfoundation/foundation/blob/main/language) meetings
+to discuss and demonstrate work using this project. 
+* Join the Open Source and Specification Working Group meetings.
+* Join the mailing lists for the [UXL Foundation](https://lists.uxlfoundation.org/g/main/subgroups) to receive meetings schedule and latest updates. 
+* Contribute to oneTBB project or oneTBB specification. Read [CONTRIBUTING](./CONTRIBUTING.md)
```
Comment (aepanchi) [inline README.md:51]:
```suggestion
* Contribute to oneTBB project or oneTBB specification. Read [CONTRIBUTING](./CONTRIBUTING.md).
```

**Review by aepanchi — APPROVED** [review]

**Review by pavelkumbrasev — APPROVED** [review]
LGTM!

**Review by pavelkumbrasev — APPROVED** [review]

**Review by aepanchi — APPROVED** [review]

**Pull request merged**